src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -205,4 +210,12 @@ export class SkyAgGridService {
   private getIconTemplate(iconName: string): string {
     return `<i class="fa fa-${iconName}"></i>`;
   }
+
+  private onKeyPress(keypress: CellKeyPressEvent) {
+    const event: KeyboardEvent = keypress.event as KeyboardEvent;
```

Comment:
You don't need both the type declaration and the cast. If `keypress.event` is declared as `any` or a type that inherits `KeyboardEvent`, you can just use `const event: KeyboardEvent = keypress.event;` Otherwise, you can use `const event = keypress.event as KeyboardEvent;`.